<launch>

    <!-- 规划程序 -->
    <node pkg="min_collision_risk_planner" name="min_collision_risk_planner_node" type="min_collision_risk_planner_node" output="screen">
        <!-- 订阅消息 -->
        <param name="odom_topic" value="/amcl_pose" />
        <param name="destination_topic" value="/move_base_simple/goal" />
        <param name="costmap_topic" value="/costmap" />
        <!-- 发布消息 -->
        <param name="init_path_topic" value="/mcrp/init_path" />
        <param name="key_vertices_topic" value="/mcrp/key_vertices" />
        <param name="processed_path_topic" value="/mcrp/processed_path" />
        <param name="optimized_path_topic" value="/global_path" />
        <param name="optimized_key_vertices_topic" value="/mcrp/optimized_key_vertices" />
        <param name="planning_result_flag_topic" value="/mcrp/planning_result_flag" />
        <param name="navigation_path_topic" value="/mcrp/navigation_path" />
        <!-- 常量 -->
        <!-- 是否保存日志信息 -->
        <param name="save_logs_flag" value="false" />
    </node>

</launch>
